tests/test_workload.py

```diff
@@ -42,7 +43,13 @@ def init_sdk(self):
         self.config = {"test1": "test1", "test2": 2}
         self.workload_result = {"1001": 12, "0110": 35, "1111": 1}
 
-    def test_create_workload(self, mock_request):
+    @pytest.fixture(autouse=True)
+    def mock_sleep(self):
+        """Fixture to mock sleep to make tests faster."""
+        with patch("time.sleep"):
```

Comment:
How does it work exactly ? How it makes tests faster ? I don't understand.

Reply:
If you patch all `time.sleep` commands, any behavior that is delayed due to a process sleeping will ignore the delay.

Specifically in this MR, 5 HTTP retries would take 31 seconds in total.

I did not write that docstr, it's a copied fixture

Reply:
Alright, thx for giving me details 👍🏻